examples/util/onnx/OnnxModelImporter.js

```diff
@@ -451,51 +451,89 @@ class OnnxModelImporter {
           opCode = this._nn.CONV_2D;
         } break;
         case 'Mul':
-        case 'Add': {
+        case 'Add':
+        case 'Sum': {
           // Add inputs
           console.log(`  inputs: [${node.input}]`);
-          const in1 = node.input[0];
-          const in2 = node.input[1];
-          inputs.push(this._getTensorIdByName(in1));
-          inputs.push(this._getTensorIdByName(in2));
 
-          const nextNode = graph.node[i+1];
-          let output = node.output[0];
-          if (nextNode && nextNode.opType === 'Relu' && node.output[0] === nextNode.input[0]) {
-            // Fuse relu
-            inputs.push(this._addScalarInt32(this._nn.FUSED_RELU));
-            i++;
-            console.log(`  fuse relu: ${nextNode.output[0]} -> ${output}`);
-            output = nextNode.output[0];
-          } else {
-            inputs.push(this._addScalarInt32(this._nn.FUSED_NONE));
+          if (node.opType === 'Sum' && node.input.length === 1) {
```

Comment:
do we need to skip `Sum` with number of inputs > 2?

Reply:
I implement this op previously when enabling [DUC segmentation](https://github.com/onnx/models/tree/master/models/semantic_segmentation/DUC). It has used multiple-input Sum. So I keep these lines for future use.

Reply:
However, current WebNN POC API doesn't support `Sum`. I understand here we have to map the `Sum` with two inputs to `Add`. Is my understanding correct?

Reply:
Yes. And I understand that I should hold it for now until we support it natively. I will create a new commit later.